homeworks/andrey.makarenko_Baksckua/3-functions/functions.js

```diff
@@ -0,0 +1,16 @@
+const OIL = 926;
+const WATER = 1000;
+const MERCURY = 1355;
+
+const numberOfTanksOil = 3;
+const numberOfTanksWeter = 5;
+const numberOfTanksMerkury = 2;
+
+const calcSumOil = OIL * numberOfTanksOil;
+const calcSumWeter = WATER * numberOfTanksWeter;
+const calcSumMerkury = MERCURY * numberOfTanksMerkury;
+
+const getWeight = function () {
+    return calcSumOil + calcSumWeter + calcSumMerkury;
+};
+console.log(getWeight);
```

Comment:
It's not even called(
You can't log a function without calling it.

`getWeight()` - called function
`getWeight` - not called function